src/wire/libevent_server.cpp

```diff
@@ -45,11 +48,14 @@ void LibeventServer::CreateNewConn(const int &connfd, short ev_flags,
                                    LibeventThread *thread,
                                    ConnState init_state) {
   auto &global_socket_list = GetGlobalSocketList();
+  recent_connfd = connfd;
```

Comment:
The updates of the member variable 'recent_connfd' are not synchronized. I am afraid that if multiple threads call the CreateNewConn() function concurrrently, the value in 'recent_connfd' could be different from the expected value.